class.jetpack-onboarding-end-points.php

```diff
@@ -539,6 +545,41 @@ static function install_theme() {
 		wp_send_json_success( $theme_id );
 	}
 
+	static function install_plugin( $slug ) {
+		if ( is_multisite() && ! current_user_can( 'manage_network' ) ) {
+			return new WP_Error( 'not_allowed', 'You are not allowed to install plugins on this site.' );
+		}
+		include_once ABSPATH . 'wp-admin/includes/class-wp-upgrader.php';
+		include_once ABSPATH . 'wp-admin/includes/file.php';
+		$upgrader  = new Plugin_Upgrader( new Automatic_Upgrader_Skin() );
+		$zip_url = "https://downloads.wordpress.org/plugin/{$slug}.latest-stable.zip";
+		$result = $upgrader->install( $zip_url );
+		if ( is_wp_error( $result ) || ! $result ) {
+			return new WP_Error( 'install_error', 'Could not install plugin ' . $slug );
+		}
+		return true;
+	}
+
+	static function install_woocommerce() {
```

Comment:
We might have to be handling the case when the user has an old WooCommerce version that does not include the setup wizard. I think the setup wizard feature was introduced in WooCommerce `2.4.0`.

Note: if we allow the user to update their WooCommerce installation, we should check whether they are allowed to do so - `current_user_can( 'update_plugins' )`.